vscode/microsoft-kiota/.vscodeignore

```diff
@@ -16,3 +16,4 @@ vsc-extension-quickstart.md
 images/samples/
 .vscode-test
 coverage/
+../
```

Comment:
this shouldn't be needed

Reply:
It is ... so that the folders at the higher level are not packaged